MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -110,13 +111,13 @@ public void setStreamType(int type) {
 
     @Override
     public void dispatch(IEvent event) {
-        if (player.get () != null && muxStats != null)
+        if (player != null && player.get () != null && muxStats != null)
```

Comment:
this was in there before, but while we're in here, change `player.get ()` to `player.get()`